src/app/parcels/parcelsTable/ParcelsPage.tsx

```diff
@@ -117,6 +120,41 @@ const ParcelsPage: React.FC<{}> = () => {
         })();
     }, []);
 
+    function getValueBoundaries<DataType>(values: (DataType | null)[]): number[] {
```

Comment:
instead of using a generic, we could just use a union type in the props, such as `values: number[] | string[]` (include nulls if you need)
We don't refer to `DataType` in the function body, so we don't need a generic